solutions/src/test/java/ru/sberbank/school/task07/FileParserImplTest.java

```diff
@@ -0,0 +1,41 @@
+package ru.sberbank.school.task07;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static ru.sberbank.school.task06.CollectionUtils.newArrayList;
+
+public class FileParserImplTest {
+
+    private final FileParser fileParser = new FileParserImpl();
+    @Ignore("Сам тест работает, проблема в билде гредла, который не видит русскую кодировку UTF-8")
```

Comment:
&#39;METHOD_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a787563764>*